en/simulation/multi_vehicle_simulation_gazebo.md

```diff
@@ -46,6 +49,53 @@ https://youtu.be/aEzFKPMEfjc
 https://youtu.be/lAjjTFFZebI
 {% endyoutube %}
 
+
+
+## Multiple Vehicles with Gazebo based on RTPS {#with_rtps}
+
+To simulate multiple vehicles based on RTPS in Gazebo, use `gazebo_sitl_multiple_run.sh` command in the terminal as described above with `-t px4_sitl_rtps` option (from the root of the *Firmware* tree).  For RTPS, the `micrortps_agent` should be run in the different terminal  for each vehicle.
+
+```
+Tools/gazebo_sitl_multiple_run.sh  -t px4_sitl_rtps [-m <model>] [-n <number_of_vehicles>] [-w <world>] [-s <script>]
+```
+
+> **Note** The model "iris_rtps" is only implemented until now.
+
+
+
+### Build and Test
+
+To build an example setup, follow the step below:
+
+1. Clone the PX4/Firmware code, then build the SITL code:
+
+```bash
+cd Firmware_clone
+git submodule update --init --recursive
+DONT_RUN=1 make px4_sitl_rtps gazebo
+```
+
+2. build `micrortps_agent`
```

Comment:
FYI, in markdown you can auto-number. You just make every item in the list `1. `. The benefit is that if you later want to add a step you don't have to renumber anything.

Reply:
You're right. Thank you for your kind comment.